gradio-app/gradio
# Adds missing parameters to `Blocks.load()`

Adds `queue` and other missing parameters to `Blocks.load()`, as well as to the documentation.

Fixes: #2714 

To test, run

```py
import gradio as gr
import time

def identity():
    time.sleep(5)
    return "hello"

with gr.Blocks() as demo:
    textbox = gr.Textbox()
    demo.load(identity, None, textbox, queue=False)
    
demo.queue().launch()
```

and open on multiple tabs. Since `queue` is disabled, you should not see a queue on the subsequent tabs and instead they should all run concurrently.

As a side note, we should refactor `Blocks.load()` altogether, as I suggest in #2754
